README.md

```diff
@@ -25,8 +25,11 @@ To fetch dependencies and get cooking:
 1. `npm install`
 2. As part of the npm install process, the `postinstall` script will install the Bower dependencies for you.
 3. Copy `config/local.json.example` to `config/local.json`, and put your local info in there.
-4. `npm start`
-5. You're up and running! surf to <http://localhost:8080> :surfer:
+4. Run `./server/db/create_db.sh` to create the database, tables, and indexes
```

Comment:
Actually, even after brew uninstalling postgresql (9.3) and reinstalling (9.4), I'm seeing this:

```sh
$ psql
psql: could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?

$ psql --version
psql (PostgreSQL) 9.4.0
```

I must go deeper.